ccl/streamingccl/streamingest: TestStreamIngestionFrontierProcessor failed

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4236151&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4236151&tab=artifacts#/) on master @ [ba04380bfbca30bf16e15048786716bec510a263](https://github.com/cockroachdb/cockroach/commits/ba04380bfbca30bf16e15048786716bec510a263):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39440086776 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
/cc @cockroachdb/bulk-io
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4237186&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4237186&tab=artifacts#/) on master @ [71becf337d9d2731298dc092f3ce9cf0f0eedb2c](https://github.com/cockroachdb/cockroach/commits/71becf337d9d2731298dc092f3ce9cf0f0eedb2c):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39440478576 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.07s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-race -parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4236824&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4236824&tab=artifacts#/) on master @ [71becf337d9d2731298dc092f3ce9cf0f0eedb2c](https://github.com/cockroachdb/cockroach/commits/71becf337d9d2731298dc092f3ce9cf0f0eedb2c):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39440440176 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4238213&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4238213&tab=artifacts#/) on master @ [71becf337d9d2731298dc092f3ce9cf0f0eedb2c](https://github.com/cockroachdb/cockroach/commits/71becf337d9d2731298dc092f3ce9cf0f0eedb2c):


```
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	github.com/cockroachdb/cockroach/pkg/rpc/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39440802504 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4241311&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4241311&tab=artifacts#/) on master @ [71becf337d9d2731298dc092f3ce9cf0f0eedb2c](https://github.com/cockroachdb/cockroach/commits/71becf337d9d2731298dc092f3ce9cf0f0eedb2c):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39440752536 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.05s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4241454&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4241454&tab=artifacts#/) on master @ [39d728b4b7ab335ff6617b250d8a53609195e07c](https://github.com/cockroachdb/cockroach/commits/39d728b4b7ab335ff6617b250d8a53609195e07c):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39441127896 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.07s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4241574&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4241574&tab=artifacts#/) on master @ [4a73be01b43b977e732bbe8f3bfc460c218a0799](https://github.com/cockroachdb/cockroach/commits/4a73be01b43b977e732bbe8f3bfc460c218a0799):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39441206688 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.05s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4246839&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4246839&tab=artifacts#/) on master @ [5b871267b62d22854f8c7b92c7c5843100597976](https://github.com/cockroachdb/cockroach/commits/5b871267b62d22854f8c7b92c7c5843100597976):


```
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	github.com/cockroachdb/cockroach/pkg/rpc/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39442562040 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4248885&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4248885&tab=artifacts#/) on master @ [5b871267b62d22854f8c7b92c7c5843100597976](https://github.com/cockroachdb/cockroach/commits/5b871267b62d22854f8c7b92c7c5843100597976):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39442634136 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.07s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4249207&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4249207&tab=artifacts#/) on master @ [75ba483e01a04e6ad707b3f0c00d645b6c92cb52](https://github.com/cockroachdb/cockroach/commits/75ba483e01a04e6ad707b3f0c00d645b6c92cb52):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39442871856 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.08s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4249292&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4249292&tab=artifacts#/) on master @ [75ba483e01a04e6ad707b3f0c00d645b6c92cb52](https://github.com/cockroachdb/cockroach/commits/75ba483e01a04e6ad707b3f0c00d645b6c92cb52):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39443204520 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4249622&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4249622&tab=artifacts#/) on master @ [323988fdbb0e7d3a3f90607811a9f657fb922f93](https://github.com/cockroachdb/cockroach/commits/323988fdbb0e7d3a3f90607811a9f657fb922f93):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39443465304 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4250315&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4250315&tab=artifacts#/) on master @ [1e965832577673d0dd3bfb11b002e562751dd376](https://github.com/cockroachdb/cockroach/commits/1e965832577673d0dd3bfb11b002e562751dd376):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39443777016 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4250737&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4250737&tab=artifacts#/) on master @ [a5158c4c93e9dca858b38ef7e94321f0a0a2b5c5](https://github.com/cockroachdb/cockroach/commits/a5158c4c93e9dca858b38ef7e94321f0a0a2b5c5):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39444053928 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4250857&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4250857&tab=artifacts#/) on master @ [a5158c4c93e9dca858b38ef7e94321f0a0a2b5c5](https://github.com/cockroachdb/cockroach/commits/a5158c4c93e9dca858b38ef7e94321f0a0a2b5c5):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39444343464 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.07s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4251097&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4251097&tab=artifacts#/) on master @ [a5158c4c93e9dca858b38ef7e94321f0a0a2b5c5](https://github.com/cockroachdb/cockroach/commits/a5158c4c93e9dca858b38ef7e94321f0a0a2b5c5):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39444597864 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.05s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-race -parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4251964&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4251964&tab=artifacts#/) on master @ [a5158c4c93e9dca858b38ef7e94321f0a0a2b5c5](https://github.com/cockroachdb/cockroach/commits/a5158c4c93e9dca858b38ef7e94321f0a0a2b5c5):


```
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	github.com/cockroachdb/cockroach/pkg/rpc/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39444833568 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4253859&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4253859&tab=artifacts#/) on master @ [a5158c4c93e9dca858b38ef7e94321f0a0a2b5c5](https://github.com/cockroachdb/cockroach/commits/a5158c4c93e9dca858b38ef7e94321f0a0a2b5c5):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39444658032 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4255227&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4255227&tab=artifacts#/) on master @ [a5158c4c93e9dca858b38ef7e94321f0a0a2b5c5](https://github.com/cockroachdb/cockroach/commits/a5158c4c93e9dca858b38ef7e94321f0a0a2b5c5):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39444979008 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4255427&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4255427&tab=artifacts#/) on master @ [a5158c4c93e9dca858b38ef7e94321f0a0a2b5c5](https://github.com/cockroachdb/cockroach/commits/a5158c4c93e9dca858b38ef7e94321f0a0a2b5c5):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39445652568 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.07s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4255496&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4255496&tab=artifacts#/) on master @ [a5158c4c93e9dca858b38ef7e94321f0a0a2b5c5](https://github.com/cockroachdb/cockroach/commits/a5158c4c93e9dca858b38ef7e94321f0a0a2b5c5):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39445964424 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.05s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4255880&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4255880&tab=artifacts#/) on master @ [a5158c4c93e9dca858b38ef7e94321f0a0a2b5c5](https://github.com/cockroachdb/cockroach/commits/a5158c4c93e9dca858b38ef7e94321f0a0a2b5c5):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39446276472 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4256176&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4256176&tab=artifacts#/) on master @ [a5158c4c93e9dca858b38ef7e94321f0a0a2b5c5](https://github.com/cockroachdb/cockroach/commits/a5158c4c93e9dca858b38ef7e94321f0a0a2b5c5):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39446603280 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4256415&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4256415&tab=artifacts#/) on master @ [a5158c4c93e9dca858b38ef7e94321f0a0a2b5c5](https://github.com/cockroachdb/cockroach/commits/a5158c4c93e9dca858b38ef7e94321f0a0a2b5c5):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39446938536 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.05s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4258691&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4258691&tab=artifacts#/) on master @ [baeba807b4c5eaecadfced511ebba0bac4a63476](https://github.com/cockroachdb/cockroach/commits/baeba807b4c5eaecadfced511ebba0bac4a63476):


```
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39447765528 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
I220131 19:21:37.869105 5421 3@vendor/github.com/cockroachdb/pebble/event.go:629  [n3,pebble,s3] 3312  [JOB 353] compacted(elision-only) L6 [000432] (1.0 K) + L6 [] (0 B) -> L6 [] (0 B), in 0.0s (0.0s total), output rate 0 B/s
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4259740&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4259740&tab=artifacts#/) on master @ [6a0bdec23578c10378766c675090dcf98d515504](https://github.com/cockroachdb/cockroach/commits/6a0bdec23578c10378766c675090dcf98d515504):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39447914928 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4261715&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4261715&tab=artifacts#/) on master @ [3e0da46a67d26cb328157038d5c4d9c0b9597d3d](https://github.com/cockroachdb/cockroach/commits/3e0da46a67d26cb328157038d5c4d9c0b9597d3d):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39448188408 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.07s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4264023&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4264023&tab=artifacts#/) on master @ [95c82a7dbc6d6fecb63edbd666c9986d1ad3885b](https://github.com/cockroachdb/cockroach/commits/95c82a7dbc6d6fecb63edbd666c9986d1ad3885b):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39448762608 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-race -parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4263569&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4263569&tab=artifacts#/) on master @ [95c82a7dbc6d6fecb63edbd666c9986d1ad3885b](https://github.com/cockroachdb/cockroach/commits/95c82a7dbc6d6fecb63edbd666c9986d1ad3885b):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39448691544 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4264954&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4264954&tab=artifacts#/) on master @ [65db9cf7d8532a539371e203e61b06bb194a8222](https://github.com/cockroachdb/cockroach/commits/65db9cf7d8532a539371e203e61b06bb194a8222):


```
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	github.com/cockroachdb/cockroach/pkg/rpc/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39449041032 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.05s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4268256&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4268256&tab=artifacts#/) on master @ [932b65f57a3f27cc8658ae7acf71706b7d7d8ea8](https://github.com/cockroachdb/cockroach/commits/932b65f57a3f27cc8658ae7acf71706b7d7d8ea8):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39448866672 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.07s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4268315&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4268315&tab=artifacts#/) on master @ [2e5cbeb1ac996db8d36ce3a714b650cdfb369058](https://github.com/cockroachdb/cockroach/commits/2e5cbeb1ac996db8d36ce3a714b650cdfb369058):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39448954200 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4268398&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4268398&tab=artifacts#/) on master @ [65db9cf7d8532a539371e203e61b06bb194a8222](https://github.com/cockroachdb/cockroach/commits/65db9cf7d8532a539371e203e61b06bb194a8222):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39449049384 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.07s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4268545&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4268545&tab=artifacts#/) on master @ [19a5d53f160593be91bd4518debc6a43ceb45562](https://github.com/cockroachdb/cockroach/commits/19a5d53f160593be91bd4518debc6a43ceb45562):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39449141400 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.05s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4268960&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4268960&tab=artifacts#/) on master @ [c4f15d6c3d6e14ab1f91849ba7c007a49f47c9a1](https://github.com/cockroachdb/cockroach/commits/c4f15d6c3d6e14ab1f91849ba7c007a49f47c9a1):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39449453544 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4270079&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4270079&tab=artifacts#/) on master @ [cb000013a625fc2eed11a50e07218e70f98fe882](https://github.com/cockroachdb/cockroach/commits/cb000013a625fc2eed11a50e07218e70f98fe882):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39449724696 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.05s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4270550&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4270550&tab=artifacts#/) on master @ [9822325761510cb8dda0b6bb0144bfc5760597b8](https://github.com/cockroachdb/cockroach/commits/9822325761510cb8dda0b6bb0144bfc5760597b8):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39449820216 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4270971&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4270971&tab=artifacts#/) on master @ [ecab73938e98e1dbcc207a06eb8e61b7acc3c119](https://github.com/cockroachdb/cockroach/commits/ecab73938e98e1dbcc207a06eb8e61b7acc3c119):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39449918352 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.05s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4272316&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4272316&tab=artifacts#/) on master @ [be1b6c4fd4778b784658459f77d28e6ed746c349](https://github.com/cockroachdb/cockroach/commits/be1b6c4fd4778b784658459f77d28e6ed746c349):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39450113136 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4274907&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4274907&tab=artifacts#/) on master @ [9f4ce34f845b6a04fa681761f0bc4544baf9ec9d](https://github.com/cockroachdb/cockroach/commits/9f4ce34f845b6a04fa681761f0bc4544baf9ec9d):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39450528144 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.07s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4276165&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4276165&tab=artifacts#/) on master @ [a434c8418c36dbeb64e73588bcd4dd5b248c3238](https://github.com/cockroachdb/cockroach/commits/a434c8418c36dbeb64e73588bcd4dd5b248c3238):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39450834864 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-race -parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4275470&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4275470&tab=artifacts#/) on master @ [63988a409584865a75ba80c0f0c3a1ceab67d076](https://github.com/cockroachdb/cockroach/commits/63988a409584865a75ba80c0f0c3a1ceab67d076):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39450705528 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.09s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4275796&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4275796&tab=artifacts#/) on master @ [16859b99637d1adedb915a67998749dbfa4c0663](https://github.com/cockroachdb/cockroach/commits/16859b99637d1adedb915a67998749dbfa4c0663):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39450796200 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4278590&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4278590&tab=artifacts#/) on master @ [a434c8418c36dbeb64e73588bcd4dd5b248c3238](https://github.com/cockroachdb/cockroach/commits/a434c8418c36dbeb64e73588bcd4dd5b248c3238):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39450875568 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.07s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4277227&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4277227&tab=artifacts#/) on master @ [a434c8418c36dbeb64e73588bcd4dd5b248c3238](https://github.com/cockroachdb/cockroach/commits/a434c8418c36dbeb64e73588bcd4dd5b248c3238):


```
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	github.com/cockroachdb/cockroach/pkg/rpc/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39451149624 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.05s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4280479&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4280479&tab=artifacts#/) on master @ [a434c8418c36dbeb64e73588bcd4dd5b248c3238](https://github.com/cockroachdb/cockroach/commits/a434c8418c36dbeb64e73588bcd4dd5b248c3238):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:89
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39451193784 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.07s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4280758&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4280758&tab=artifacts#/) on master @ [0e5eb08368020e7633142fd60f0f818fcd46510d](https://github.com/cockroachdb/cockroach/commits/0e5eb08368020e7633142fd60f0f818fcd46510d):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:110
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39451413936 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4281204&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4281204&tab=artifacts#/) on master @ [8a76b5916e5f6367572f1b2413704f82d82f3da6](https://github.com/cockroachdb/cockroach/commits/8a76b5916e5f6367572f1b2413704f82d82f3da6):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:110
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39451581528 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.07s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4281630&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4281630&tab=artifacts#/) on master @ [39ca6aca0806fa1dc1e846c7014799fb71621587](https://github.com/cockroachdb/cockroach/commits/39ca6aca0806fa1dc1e846c7014799fb71621587):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:110
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39451665480 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4282273&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4282273&tab=artifacts#/) on master @ [d8693413c0149060bbf16353dd29bff7dae6f64c](https://github.com/cockroachdb/cockroach/commits/d8693413c0149060bbf16353dd29bff7dae6f64c):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:110
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39451794456 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.10s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4283059&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4283059&tab=artifacts#/) on master @ [577e9b0e078f66b4a0d6c8c73e3aa6c7bba297ec](https://github.com/cockroachdb/cockroach/commits/577e9b0e078f66b4a0d6c8c73e3aa6c7bba297ec):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:110
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:190
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2021
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1569
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:910
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:893
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:803
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39451884168 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4283974&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4283974&tab=artifacts#/) on master @ [a0f0df792704250649a09bbbbfa89c0a0ce1e75a](https://github.com/cockroachdb/cockroach/commits/a0f0df792704250649a09bbbbfa89c0a0ce1e75a):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:110
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:201
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2027
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1575
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:913
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:896
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:806
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39452051832 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4284428&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4284428&tab=artifacts#/) on master @ [88522a3d6346e8302ce7a027b8b6a9184828ef13](https://github.com/cockroachdb/cockroach/commits/88522a3d6346e8302ce7a027b8b6a9184828ef13):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:110
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:201
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2027
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1575
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:913
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:896
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:806
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39452152152 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4286790&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4286790&tab=artifacts#/) on master @ [a950be5e3399abfb02a05b665fd81b965cc58d11](https://github.com/cockroachdb/cockroach/commits/a950be5e3399abfb02a05b665fd81b965cc58d11):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:110
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:201
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2027
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1575
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:913
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:896
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:806
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39452549904 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4288601&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4288601&tab=artifacts#/) on master @ [f20c1a8d824f0d00f18c0d58ed8b4bc4955e0969](https://github.com/cockroachdb/cockroach/commits/f20c1a8d824f0d00f18c0d58ed8b4bc4955e0969):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:110
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:201
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2027
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1575
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:913
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:896
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:806
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39452927880 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.05s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-race -parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4288265&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4288265&tab=artifacts#/) on master @ [f20c1a8d824f0d00f18c0d58ed8b4bc4955e0969](https://github.com/cockroachdb/cockroach/commits/f20c1a8d824f0d00f18c0d58ed8b4bc4955e0969):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:110
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:201
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2027
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1575
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:913
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:896
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:806
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39452905008 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4289509&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4289509&tab=artifacts#/) on master @ [5c0d5132ade6d2e30a9d414163851272750d8afa](https://github.com/cockroachdb/cockroach/commits/5c0d5132ade6d2e30a9d414163851272750d8afa):


```
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_raft.go:110
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/store_send.go:201
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	github.com/cockroachdb/cockroach/pkg/rpc/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2027
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1575
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:913
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:896
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	github.com/cockroachdb/cockroach/pkg/kv/db.go:806
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39453232488 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.05s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4292921&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4292921&tab=artifacts#/) on master @ [5c0d5132ade6d2e30a9d414163851272750d8afa](https://github.com/cockroachdb/cockroach/commits/5c0d5132ade6d2e30a9d414163851272750d8afa):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:110
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:201
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2027
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1575
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:913
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:896
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:806
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39453068832 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.07s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4293042&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4293042&tab=artifacts#/) on master @ [5c0d5132ade6d2e30a9d414163851272750d8afa](https://github.com/cockroachdb/cockroach/commits/5c0d5132ade6d2e30a9d414163851272750d8afa):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:110
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:201
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2027
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1575
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:913
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:896
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:806
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39453405768 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.08s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4293046&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4293046&tab=artifacts#/) on master @ [61e2304e5c0fc448942a8659af129851b34a00f9](https://github.com/cockroachdb/cockroach/commits/61e2304e5c0fc448942a8659af129851b34a00f9):


```
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_proposal.go:889
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).evalAndPropose
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:110
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeWriteBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_write.go:153
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeBatchWithConcurrencyRetries
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:517
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithoutRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:247
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:96
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:201
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:191
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:957
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:344
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:945
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:1014
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:565
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:209
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:191
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.raceTransport.SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport_race.go:79
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:2027
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1575
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1177
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:798
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:221
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:913
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:896
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:806
        Wraps: (7) SST stats are incorrect: diff(given, actual) = [GCBytesAge: 39453464112 != 0]
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.withPrefix (5) *assert.withAssertionFailure (6) *errbase.opaqueWrapper (7) *errutil.leafError TraceData:[] LeafTxnFinalState:<nil> RowNum:<nil> SamplerProgress:<nil> BulkProcessorProgress:<nil> Metrics:<nil>}
    --- FAIL: TestStreamIngestionFrontierProcessor/no-checkpoints (0.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/kv/kvserver/batcheval/cmd_add_sstable.go
+++ b/pkg/kv/kvserver/batcheval/cmd_add_sstable.go
@@ -317,11 +317,13 @@ func assertSSTContents(sst []byte, sstTimestamp hlc.Timestamp, stats *enginepb.M
 		iter.Next()
 	}
 
-	// Compare statistics with those passed by client.
+	// Compare statistics with those passed by client. We calculate them at the
+	// same timestamp as the given statistics, since they may contain
+	// timing-dependent values (typically MVCC garbage, i.e. multiple versions).
 	if stats != nil {
 		given := *stats
-		given.LastUpdateNanos = 0
-		actual, err := storage.ComputeStatsForRange(iter, keys.MinKey, keys.MaxKey, 0)
+		actual, err := storage.ComputeStatsForRange(
+			iter, keys.MinKey, keys.MaxKey, given.LastUpdateNanos)
 		if err != nil {
 			return errors.Wrap(err, "failed to compare stats: %w")
 		}

